andela/AndelaEatsUI
# #161146390 Remove auto-bind module

#### What does this PR do?
- Removes auto-bind npm package from project in favor of defining  class instance methods with arrow function.

#### Description of Task to be completed?
- Uninstall autoBind module
- Remove autoBind module from code
- Turn instance methods to arrow functions

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ch-remove-autobind-161146390`
- Run these commands `npm run bundle` and `npm run dev` to setup app
- Log in as an `Admin`, navigate to `admin dashboard`, select `Meals Icon` on the `side nav`.
- Ensure the meals pages works as expected.
- Run the command `npm run test-dev` to ensure all tests are passing.

#### What are the relevant pivotal tracker stories?
- [#161146390](https://www.pivotaltracker.com/story/show/161146390)
